src/pybind/mgr/cephadm/utils.py

```diff
@@ -21,7 +21,7 @@ class CephadmNoImage(Enum):
 
 # ceph daemon types that use the ceph container image.
 # NOTE: order important here as these are used for upgrade order
-CEPH_TYPES = ['mgr', 'mon', 'crash', 'osd', 'mds', 'rgw', 'rbd-mirror', 'cephfs-mirror']
+CEPH_TYPES = ['mgr', 'mon', 'crash', 'osd', 'mds', 'rgw', 'rbd-mirror', 'cephfs-mirror', 'ceph-exporter']
```

Comment:
@avanthakkar can we have this `ceph-exporter` in MONITORING_STACK_TYPES as it's intended for Prometheus metrics purpose?
also what version does `ceph-exporter` gets assigned?

Reply:
@sunilangadi2 It's already added https://github.com/ceph/ceph/pull/46213/files#diff-d0f7acffbce59b9e36a1479d1b1f32955cd811ab8b0b1aa0018a2003f0c4d73eR5262

Reply:
it's a ceph daemon so it will have ceph version